test/verify/check-static-login

```diff
@@ -414,14 +414,19 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
     def testRaw(self):
         self.machine.start_cockpit()
         time.sleep(0.5)
+        http_auth_denied_code = 401
+        # On Arch Linux PAM errors are treated a bit different.
+        if self.machine.image == "arch":
+            http_auth_denied_code = 403
+
         self.assertEqual(self.curl_auth_code('/cockpit/login', ''), 401)
         self.assertEqual(self.curl_auth_code('/cockpit/login', 'foo:'), 401)
         self.assertEqual(self.curl_auth_code('/cockpit/login', 'foo:bar\n'), 401)
         self.assertEqual(self.curl_auth_code('/cockpit/login', 'foo:bar:baz'), 401)
         self.assertEqual(self.curl_auth_code('/cockpit/login', ':\n\n'), 401)
-        self.assertEqual(self.curl_auth_code('/cockpit/login', 'admin:bar'), 401)
+        self.assertEqual(self.curl_auth_code('/cockpit/login', 'admin:bar'), http_auth_denied_code)
         self.assertEqual(self.curl_auth_code('/cockpit/login', 'foo:bar'), 401)
```

Comment:
Similarly to the previous comment on an earlier commit, using `assertIn(..., [401, 403])` seems perfectly fine here. Also, these two changes should be in the same commit.